index.html

```diff
@@ -81,6 +81,9 @@
             <li class="header__nav-item">
               <a title="About us" class="header__nav-link" href="about-us.html">About us</a>
             </li>
+            <li class="header__nav-item">
+              <a title="Projects" class="header__nav-link" href="index.html">Projects</a>
```

Comment:
This can be done when project page (project.html) is created.
So no need to add this right now.